app/admin/users.rb

```diff
@@ -23,6 +27,12 @@
     end
   end
 
+  member_action :invite, only: [:post] do
+    UserMailer.with(user: resource).invite.deliver_later
```

Comment:
What does the `with` method do? I'm having a terrible time googling it.  Looks like it sets a params variable? Why is this preferred over passing an argument to  the `invite` method?

Reply:
Passing an argument is how I'd always done it, too. I just noticed the example is different in the current actionmailer docs so I went with the latest style that didn't take a param and used `with`:

https://guides.rubyonrails.org/action_mailer_basics.html#edit-the-mailer
https://guides.rubyonrails.org/action_mailer_basics.html#calling-the-mailer